INSTALL

```diff
@@ -615,7 +615,7 @@ The directories set up by Configure fall into three broad categories.
 
 =item Directories for the perl distribution
 
-By default, Configure will use the following directories for 5.32.0.
+By default, Configure will use the following directories for 5.33.0.
```

Comment:
perl7? (a few more cases in this file)

Reply:
indeed going to make these changes, was concerned about the noise of such a change in a first iteration 
but will do this in a later commit